README.md

```diff
@@ -58,7 +58,7 @@ or if available, just make your usual PostgreSQL user a `SUPERUSER`.
 
 Or you can use the [PGXN Extension Build and Test Tools](https://github.com/pgxn/docker-pgxn-tools) Docker image:
 `docker run -it --rm --mount "type=bind,src=$(pwd),dst=/repo" pgxn/pgxn-tools \
-  sh -c 'cd /repo && pg-start 13 && pg-build-test'`.
+  sh -c 'cd /repo && apt update && apt install -y jq && pg-start 13 && pg-build-test'`.
```

Comment:
The `\` should be removed because inline snippets will collapse to single line.

Reply:
OK, so with the `\` it doesn't work, if copied from rendered MD. Without the `\` it is only slightly weird and inconvenient, so without the `\` wins...

Or we can just not have the newline there at all, that would work well in both, just a long line. To me this wins.